src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -202,45 +205,6 @@ private void WriteToStringOverride(INamedTypeSymbol typeSymbol, IndentedStringBu
 				}
 			}
 
-			private void WriteiOSMoveToSuperView(INamedTypeSymbol typeSymbol, IndentedStringBuilder builder)
-			{
-				var isiosView = typeSymbol.Is(_iosViewSymbol);
```

Comment:
Since we error if we have ios view, this is going to always be false. Meaning that this method will never generate any code. So I deleted it altogether. Let me know if I'm misunderstanding something here.

Reply:
We error out only outside of the compilation for uno itself. Uno needs this.

Reply:
Does the source generator already have the concept to differential whether we're in Uno compilation or outside? or this should be a "new concept"?

Reply:
You can check if the current solution is `Uno.UI.sln`, that should be enough for now.

Reply:
@jeromelaban I don't know of a way that source generators can access this kind of information. We could introduce a MSBuild property in `Directory.Build.props` which says whether we're in Uno solution. I'll add that, but let me know if you know of a better way. Thanks!

Reply:
It's accessibly from generators, you need to add `SolutionFileName` to [this block](https://github.com/unoplatform/uno/blob/5e7cb23acf49113cbfc56d70c46f5f37dc0ff6bc/src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props#L65), then access it through something [similar to this](https://github.com/unoplatform/uno/blob/5e7cb23acf49113cbfc56d70c46f5f37dc0ff6bc/src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/DependencyObjectAvailabilityGenerator.cs#L78).